src/Workspaces/Core/MSBuild/WorkspaceMSBuildResources.resx

```diff
@@ -133,10 +133,16 @@
     <value>Invalid {0} specified: {1}</value>
   </data>
   <data name="Msbuild_failed_when_processing_the_file_0" xml:space="preserve">
-    <value>Msbuild failed when processing the file '{0}'</value>
+    <value>MSBuild failed when processing the file '{0}'</value>
   </data>
   <data name="Msbuild_failed_when_processing_the_file_0_with_message_1" xml:space="preserve">
-    <value>Msbuild failed when processing the file '{0}' with message: {1}</value>
+    <value>MSBuild failed when processing the file '{0}' with message: {1}</value>
+  </data>
+  <data name="Msbuild_warning_when_processing_the_file_0" xml:space="preserve">
+    <value>MSBuild warning when processing the file '{0}'</value>
```

Comment:
I think we should change this string to `MSBuild warned ...` to keep it inline with the `MSBuild failed ...` string.

Reply:
Done.